tutorials/math/random_number_generation.rst

```diff
@@ -196,6 +333,54 @@ We can apply similar logic from arrays to dictionaries as well::
         # The same metal may be selected multiple times in succession.
         return random_metal
 
+ .. code-tab:: csharp
+
+    //To get access to .ToArray() on the dictionary
+    using System.Linq;
```

Comment:
I couldn't pull off the C# equivalent without `Sysyem.Linq`. Correct me if I'm wrong in using this.